templates/cluster-template-managed-private.yaml

```diff
@@ -78,8 +78,13 @@ metadata:
   name: ${CLUSTER_NAME}-mp-0
   namespace: default
 spec:
+  version: "${KUBERNETES_VERSION}"
```

Comment:
thought: what about `kubeVersion` or `kubernetesVersion` or something? I don't love the length of it but it is much more clear that way. 🤷